opam.in

```diff
@@ -32,4 +32,4 @@ depopts: [
 conflicts: [
   "utop" {< "1.17"}
 ]
-available: [ ocaml-version = "4.02.3" ]
+available: [ ocaml-version = "4.04.0" ]
```

Comment:
This is a bit incorrect... Anything from 4.02.3 to 4.05.0 will do.
Actually, it is available as long as ocaml-migrate-parsetree is.